manifests/init.pp

```diff
@@ -820,8 +820,77 @@
                 ]
               }
             }
+            '16.04': {
+              $default_pam_d_login_template = 'pam/login.ubuntu16.erb'
+              $default_pam_d_sshd_template  = 'pam/sshd.ubuntu16.erb'
+              $default_package_name         = 'libpam0g'
+
+              if $ensure_vas == 'present' {
+                $default_pam_auth_lines = [
```

Comment:
If this is just copy/paste and has not been tested with VAS, then we should just fail() and say that Ubuntu 16.04 does not support VAS. I would rather do that than assume it works.